webapp/templates/jury/submission.html.twig

```diff
@@ -778,6 +778,12 @@
                             {% else %}
                                 <pre class="output_text">{{ runsOutput[runIdx].output_system }}</pre>
                             {% endif %}
+
+                            {% if runsOutput[runIdx].team_message is not empty %}
+                                <h5>Judge message</h5>
```

Comment:
I would even add `for team` here, since for jury the judge message is more known than to teams. Maybe we don't need it for the team pages, only jury pages? cc @meisterT WDYT?

Reply:
agreed